rq/worker.py

```diff
@@ -1089,14 +1089,14 @@ def handle_job_success(self, job: 'Job', queue: 'Queue', started_job_registry: S
 
     def execute_success_callback(self, job: 'Job', result):
         """Executes success_callback with timeout"""
-        self.log.debug("Running Success Callbacks.")
+        self.log.debug(f"Running success callbacks for {job.id}")
         job.heartbeat(utcnow(), CALLBACK_TIMEOUT)
         with self.death_penalty_class(CALLBACK_TIMEOUT, JobTimeoutException, job_id=job.id):
             job.success_callback(job, self.connection, result)
 
     def execute_failure_callback(self, job):
         """Executes failure_callback with timeout"""
-        self.log.debug("Running Failure Callbacks.")
+        self.log.debug(f"Running Failure Callbacks for {job.id}")
```

Comment:
Nitpick: `failure callbacks` should not have uppercase as first letters :)

Reply:
"Should" is a strong word to use :) But got your point will change that.